include/circt/Dialect/Calyx/CalyxControl.td

```diff
@@ -372,3 +372,38 @@ def StaticRepeatOp : CalyxContainer<"static_repeat", [
   ];
 
 }
+
+def RepeatOp : CalyxContainer<"repeat", [
```

Comment:
Should this share an interface, similar `IfOpInterface` for the dynamic and static variants?

Reply:
I'm not quite sure what would go in the `RepeatInterface` (although to be clear, I'm not that familiar with how the codebase works). 

It seems like `IfInterface` mainly is used for a verifier, but verifying the static/dynamic repeats is different. Static just checks if the body is static, and there is no verifier for dynamic (which I thought was fine, given that dynamic `seq`  and `par` blocks also don't have verifiers).

Reply:
It seems like there isn't much to be shared in this case, so I think it would be fine to not have an interface. The canonicalizations are templated, which is another way to handle reuse.

Reply:
Ok makes sense, perhaps I'm being unnecessarily preemptive :)